native-gradle-plugin/src/main/java/org/graalvm/buildtools/gradle/NativeImagePlugin.java

```diff
@@ -131,6 +131,7 @@
      */
     private static final Consumer<Object> FORCE_CONFIG = t -> {
     };
+    private static final String JUNIT_PLATFORM_LISTENER_ENABLE = "junit.platform.listeners.uid.tracking.enabled";
```

Comment:
```suggestion
    private static final String JUNIT_PLATFORM_LISTENERS_UID_TRACKING_ENABLED = "junit.platform.listeners.uid.tracking.enabled";
```

For consistency with `NativeExtension`.